Sources/Config.swift

```diff
@@ -1,4 +1,12 @@
 extension Warp {
+    public static var Token: TokenProvider {
+        return Config.tokenProvider
```

Comment:
Maybe just copy and paste that code to here and just deprecate the ones inside Config so other people stop using those too 👍

Reply:
My plan was to get this into a new release.
Fix the NMP project and then deprecate/remove the old variables in a new Warp release after.
Just to ensure the project always can compile. But again. we have a summer release freeze in NMP, so maybe we should just make the cut? 🤔

Reply:
I say just go for it 😅 should be fine. It is just a deprecation so only warnings even if someone is using it.

Reply:
Looks great 🥳